src/index.js

```diff
@@ -0,0 +1,167 @@
+// 'use strict';
+import 'regenerator-runtime/runtime';
+import axios from 'axios';
+
+const state = {
+  tempCount: 60,
+  cityName: 'Seattle',
+  latitude: 0,
+  longitude: 0,
+};
+
+function convertTempKtoF(tempInK) {
+  return Math.round(1.8 * (tempInK - 273) + 32);
+}
+
+const getCityCoordinates = () => {
+  axios
+    .get('https://fast-dawn-95769.herokuapp.com/location', {
+      params: {
+        q: state.cityName,
+      },
+    })
+    .then((response) => {
+      state.latitude = response.data[0].lat;
+      state.longitude = response.data[0].lon;
+      console.log('Success: ', response.data);
```

Comment:
Feel free to omit console logs after debugging - it makes for a cleaner console for users to see errors or developer messages.